.openshift-ci/get_latest_helm_chart_versions.py

```diff
@@ -64,6 +69,13 @@ def main(argv):
         f"Latest chart version for the {sample_support_exception} "
         f"releases is {helm_version_specific}"
     )
+    supported_versions_string = [[f"{version.major}.{version.minor}"] for version in get_supported_releases()]
```

Comment:
Note: I can't comment outside of the changed lines.

[nit] I know this was not changed in this PR but I saw a `print` in line 64. I think we should changed to `logging`.

Reply:
Okay, changing it. It's not that important since it's only called within main() which is used to debug the script / check if it's working correctly but it's better nonetheless.